src/main/java/school/lemon/changerequest/java/introduction/hw1/Task6.java

```diff
@@ -2,10 +2,28 @@
 
 public class Task6 {
     public static double calculateS(double x) {
-        return 0d;
+        double s;
+        s = 1 + x + Math.pow(x, 2) / factorial(2) + Math.pow(x, 3) / factorial(3) + Math.pow(x, 4) / factorial(4);
```

Comment:
What if you'll need to calculate 10 more terms ?
For example: `x^2/2!+x^3/3!+...+x^10/10!`.
Could you please refactor your code to be more generic ?